app/controllers/application_controller.rb

```diff
@@ -15,7 +15,7 @@ class ApplicationController < ActionController::Base
 
   impersonates :user
 
-  if Rails.env.development? && ENV["REDESIGN"] == "true"
+  if ENV["REDESIGN"] == "true"
```

Comment:
Note that I removed the development check so we can start testing this on staging.